### DISCUSSION TITLE
这个致谢这里怎么往下挪动啊

### DISCUSSION THREAD

**Discussion by RingKing1** [TheNetAdmin/zjuthesis#334] [General]
```
\cleardoublepage
\vskip 10pt
\chapternonum{致谢}
```
我使用这样的代码，但是没有效果
![image](https://github.com/TheNetAdmin/zjuthesis/assets/45783443/fb4efe52-8ae8-4576-a068-5898e54952f2)

**Comment by RingKing1** [comment]
![image](https://github.com/TheNetAdmin/zjuthesis/assets/45783443/f81074a1-207a-48ea-a48a-ddd6f3371963)
这种情况是正确的，但是不居中

**Comment by RingKing1** [comment]
` \begin{center}
            \vspace*{0.2em}
            \bfseries \zihao{3} #1
 \end{center}`
我先贴一个我的方法，但是感觉不太对，间距和目录那边还是有点不一样。

**Comment by chengzhongyi8** [comment]
请问解决了吗？？

**Comment by RingKing1** [comment]
就是我上面提到的方法，没有其他更好的了

**Comment by chengzhongyi8** [comment]
效果如下，感觉上下有一点点的不一样

<img width="458" alt="图片" src="https://github.com/TheNetAdmin/zjuthesis/assets/58238523/4a8e87d1-ab1f-4987-bc3c-631b0287a30b">

**Reply by RingKing1** [reply]
你可以尝试用vspace空两行
